pgpy/pgp.py

```diff
@@ -85,6 +85,7 @@
 
 
 class PGPSignature(Armorable, ParentRef, PGPObject):
+    _reason_for_revocation = collections.namedtuple('ReasonForRevocation', ['code', 'comment'])
```

Comment:
i'm not sure this pattern is common elsewhere in the PGPy code. if it isn't, i would suggest describing the idea behind it a little more here, since it stumped me for a while - at first glance i thought this was a class-static variable/cache.

Reply:
I did it that way since [types.py->SignatureVerification](https://github.com/SecurityInnovation/PGPy/blob/master/pgpy/types.py#L582) does the same thing.

Reply:
ah yes, i noticed that later, actually quite elegant once i wrap my head around it. :)